http/tus_handlers.go

```diff
@@ -150,6 +157,13 @@ func tusPatchHandler() handleFunc {
 
 		w.Header().Set("Upload-Offset", strconv.FormatInt(uploadOffset+bytesWritten, 10))
 
+		if uint64(bytesWritten) < (d.Settings.Tus.ChunkSize) {
+			err = d.RunAfterHook("upload", r.URL.Path, "", d.user)
+			if err != nil {
+				return errToStatus(err), err
+			}
+		}
+
```

Comment:
What happens if the file size is exactly a multiple of the chunk size? I think then the after hook won't run.